### ISSUE TITLE
Ensure meta descriptions are at least 60 characters long

### ISSUE THREAD

**Issue by eons44** [web-infrastructure/doc_architecture#6]
Meta descriptions below 120 characters are generally failing to take full advantage of available space in a search engine results page.

Short descriptions are typically ignored or expanded by Google with content from the rest of your page. This usually results in suboptimal text that is less likely to be clicked.

You can use your discretion here in ignoring pages which are not sufficiently important. Generally landing pages and pages which could feature prominently in search engine results should aim to keep within these limits.
